README.md

```diff
@@ -2,5 +2,46 @@
 
 ## Objectives
 
++ Link jQuery to HTML file
 + Write inline jQuery to modify HTML
-+ Explain how jQuery modifies HTML
\ No newline at end of file
++ Explain how jQuery modifies HTML
+
+
+## Inline jQuery
+
+We're going to use jQuery to add some text to our HTML page. 
+
+### Include jQuery Link
+
+In order to start writing jQuery, we need to include the library in our HTML. You'll want to copy the code below and paste it `html/index.html` in between your head tags. This script tag links our HTML file to the jQuery library.
+
+```html
+<script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.3/jquery.min.js"></script>
```

Comment:
maybe explain the difference between downloading the library vs using a hosted CDN?